addons/io_scene_gltf2/blender/exp/gltf2_blender_gather_materials.py

```diff
@@ -111,13 +111,30 @@ def __gather_emissive_factor(blender_material, export_settings):
     if emissive_socket is None:
         emissive_socket = gltf2_blender_get.get_socket_old(blender_material, "EmissiveFactor")
     if isinstance(emissive_socket, bpy.types.NodeSocket):
-        fac = gltf2_blender_get.get_factor_from_socket(emissive_socket, kind='RGB')
-        if fac is None and emissive_socket.is_linked:
+        factor = gltf2_blender_get.get_factor_from_socket(emissive_socket, kind='RGB')
+
+        if factor is None and emissive_socket.is_linked:
             # In glTF, the default emissiveFactor is all zeros, so if an emission texture is connected,
             # we have to manually set it to all ones.
-            fac = [1.0, 1.0, 1.0]
-        if fac == [0, 0, 0]: fac = None
-        return fac
+            factor = [1.0, 1.0, 1.0]
+
+        # Handle Emission Strength
+        strength_socket = None
+        if emissive_socket.node.type == 'EMISSION':
+            strength_socket = emissive_socket.node.inputs['Strength']
+        elif 'Emission Strength' in emissive_socket.node.inputs:
+            strength_socket = emissive_socket.node.inputs['Emission Strength']
```

Comment:
These two lines should support [D4971](https://developer.blender.org/D4971), but I haven't tested it or anything.